draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -132,31 +132,34 @@
       the Metadata about 5G low latency edge services directly attached to the
       egress routers. 5G <xref target="TS.23.501-3GPP"/>is characterized by
       having edge services closer to the Cell Towers reachable by Local Data
-      Networks (LDN). From IP network perspective, the 5G LDN is a limited
+      Networks (LDN). From an IP network perspective, the 5G LDN is a limited
       domain <xref target="RFC8799"/> with edge services a few hops away from
       the ingress nodes. Only selective UE services are considered as 5G low
       latency edge services.</t>
 
       <t>Note: The proposed edge service Metadata Path Attribute are not
-      intended for the best-effort services reachable via the public internet.
+      intended for the best-effort services reachable via the public Internet.
       The information carried by the Metadata Path Attribute can be used by
       the ingress routers to make path selections for selective low latency
       services based on not only the network distance but also the running
       environment of the edge cloud sites. The goal is to improve latency and
       performance for 5G ultra-low latency services.</t>
 
-      <t>The extension is targeted for a single domain with RR controlling the
-      propagation of the BGP UPDATE. The edge service Metadata Path Attribute
+      <t>This extension is targeted for a single domain with a BGP Route
+      Reflector (RR) <xref target="RFC4456"/>  controlling the
+      propagation of the BGP UPDATEs. The edge service Metadata Path Attribute
       is only attached to the low latency services (routes) hosted in the 5G
-      edge cloud sites, which are only a small subset of services initiated
+      edge cloud sites. These routes are only a small subset of services initiated
       from UEs, not for UEs accessing many internet sites.</t>
 
       <t>While the proposed Metadata Path Attribute is particularly beneficial
-      for low latency services, the metadata path attributes can be expanded
+      for low latency services, the Metadata Path Attributes can be expanded
       to propagate information about GPU availability, power, or other
       resources necessary for compute-intensive services such as AI and
       machine learning. This flexibility makes it a valuable tool for a wide
       range of applications beyond just low latency services.</t>
+      <!-- XXX JMH - this paragraph hints "we can do more with this".  However,
+      it also opens the proposal outside of the limited domain targeting. -->
     </section>
```

Comment:
Is it necessary to add the following sentence stating additional are beyond the scope of this document?

_"However, exploring the full potential of these expanded use cases is beyond the scope of this document."_

Reply:
Some of the things in this document completely lose their safety (for example, the inconsistent route selection) the minute you try to use this in a different environment.

Your additional sentence doesn't address a need to enforce that lack of general purpose safety.

Reply:
Can adding the following sentence address your comment? 

_"To ensure consistent route selection and avoid inconsistencies across different ingress nodes, it is essential to use a consistent algorithm for processing the Metadata Path Attribute. This will help guarantee that all ingress nodes interpret and use the Metadata information in the same manner, preventing potential conflicts and enhancing the overall network performance."_

Reply:
I added a few words noting that this is for limited domain network use. That addresses my immediate concerns.